src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -81,11 +82,13 @@ public static void main(String[] args) {
 
     DLPTextToBigQueryStreamingV2PipelineOptions options =
         PipelineOptionsFactory.fromArgs(args).as(DLPTextToBigQueryStreamingV2PipelineOptions.class);
+    Util.validateBQStorageApiOptionsStreaming(options);
```

Comment:
Were these changes tested for both exactly once and at least once semantics?

Reply:
Yes both of these cases have been tested. Thanks will mention the same in PR Description as well